data/spiders/rips/missing_info_page.html

```diff
@@ -0,0 +1,154 @@
+<html>
+<head></head>
```

Comment:
Why do you need to store HTML in git?

Reply:
I think the HTML file is a good fixture for Unit Tests.

Reply:
Oh, I see, if it's for tests it should be somewhere under the `tests` directory